src/Microsoft.ML.DnnImageFeaturizer.AlexNet/AssemblyPathHelpers.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.ML.Transforms
+{
+    internal static class AssemblyPathHelpers
+    {
+        public static string GetExecutingAssemblyLocation()
+        {
+            string codeBaseUri = Assembly.GetExecutingAssembly().CodeBase;
+            string path = new Uri(codeBaseUri).AbsolutePath;
```

Comment:
Take a look at @JeremyKuhne's advice here about using `Uri`s for local file paths.

[https://github.com/dotnet/core-setup/issues/4969#issuecomment-456923839](https://github.com/dotnet/core-setup/issues/4969#issuecomment-456923839)

Reply:
i think we should be okay with using uri here because here we are not constructing a uri against a path.
The codebase already returns a uri in form of a string. I am converting it back to uri to get the local path from the uri.